config.example.yaml

```diff
@@ -15,54 +20,21 @@ adapters:
      adapter: "@mate/adapter-squid"
      config:
         integratorId: mate-sdk
-        baseUrl: https://api.0xsquid.com
+        baseUrl: https://testnet.api.squidrouter.com
 flows:
-   - name: USD to cUSD
-     id: usd-to-cust-binance
-     description: "Buy USDC on Binance -> Withdraw to Ethereum -> Bridge USDC from Ethereum to axlUSDC on Celo using Squid"
-     steps:
-        - type: Exchange.Swap
-          adapter: ccxt
-          config:
-             exchange: binance
-             base: USDC
-             quote: USD
-             side: Buy
-             maxSlippage: 20
-             amount: 1000
-        - type: Exchange.WithdrawCrypto
-          adapter: ccxt
-          config:
-             exchange: binance
-             asset: USDC
-             chainId: 1
-             destinationAddress: 0xMyAddress1
-             amount: 500_000
-        - type: Bridge.Swap
-          adapter: squid
-          config:
-             fromChain: 1
-             fromToken: 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48
-             fromAddress: 0xMyAddress1 -> TODO Remove and use primary address as default
-             fromAmount: { stepOutput.ExchangeWithdrawCrypto.0.amount }
-             toChain: 42220
-             toToken: 0x765DE816845861e75A25fCA122bb6898B8B1282a
-             toAddress: 0xMyAddress1
-             maxSlippage: 0.01
-   - name: axlUSDC to USD
-     id: axlusd-to-usd
-     description: "Bridge axlUSDC on Celo to USDC on Ethereum -> Deposit USDC to Binance Exchange -> Swap USDC to USD on Binance"
+   - name: aUSDC to aUSDC
+     id: auscd-to-ausdc
+     description: "Bridge aUSDC on Goerli to aUSDC on Celo"
      steps:
         - type: Bridge.Swap
           adapter: squid
           config:
-             fromChain: 42220
-             fromToken: 0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48
-             fromAddress: 0xMyAddress1 -> TODO Remove and use primary address as default
-             fromAmount: { stepOutput.ExchangeWithdrawCrypto.0.amount }
-             toChain: 1
-             toToken: 0x765DE816845861e75A25fCA122bb6898B8B1282a
-             toAddress: 0xMyAddress1
+             fromChain: 5
+             fromToken: "0x254d06f33bDc5b8ee05b2ea472107E300226659A"
```

Comment:
```suggestion
             # https://goerli.etherscan.io/token/0x254d06f33bDc5b8ee05b2ea472107E300226659A
             fromToken: "0x254d06f33bDc5b8ee05b2ea472107E300226659A"
```

Reply:
I'm on the line with this, from my perspective this doesn't add much. How do you think this helps?

Reply:
just a habit of mine that i always like to double-check addresses and the easier I make it for myself the more likely I'm going to do it

not feeling strongly about this to add it here :)

Reply:
In the grand scheme of things, we're still at a low level here. When we bring in a CLI we can implement further checks on these addresses to prevent user error. We could pull in token information based on the chain and address and get them to confirm before proceeding.

Reply:
I am going back on this. It's useful for sanity checking. 👍